packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1016,7 +1016,8 @@ contract Deploy is Deployer {
                         disputeGameFactory: mustGetAddress("DisputeGameFactoryProxy"),
                         optimismPortal: mustGetAddress("OptimismPortalProxy"),
                         optimismMintableERC20Factory: mustGetAddress("OptimismMintableERC20FactoryProxy"),
-                        gasPayingToken: customGasTokenAddress
+                        gasPayingToken: customGasTokenAddress,
+                        l2ETHToken: address(0)
```

Comment:
I see we are going with l2ETHToken = addr(0) for the tests. So the two possible values of l2ETHToken are either addr(0) for Boba-ETH and L2BNB for Boba-BNB right?

Reply:
Another unit tests are added. For the local deployment we assume that this l2ETHToken is disabled.